src/components/ItemsList.js

```diff
@@ -3,15 +3,24 @@ import firebase from '../lib/firebase';
 import Nav from './Nav';
 import '../styles/ItemsList.css';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
+import calculateEstimate from '../lib/estimates';
 import { useHistory } from 'react-router-dom';
 import '../styles/ItemsList.css';
 
 const db = firebase.firestore().collection('shopping_list');
+const wasItemPurchasedWithinLastOneDay = (purchaseDates) => {
+  if (!purchaseDates.length) return false;
+  const oneDayInMilliseconds = 24 * 60 * 60 * 1000;
+  const { length } = purchaseDates;
+  return Date.now() - purchaseDates[length - 1] <= oneDayInMilliseconds;
+};
 
-const wasItemPurchasedWithinLastOneDay = (lastPurchasedOn) => {
-  if (lastPurchasedOn === null) return false;
+const getDaysBetweenCurrentAndPreviousPurchase = (
+  previousPurchaseDate,
```

Comment:
This is pretty cool, that you changed it to a string to store all the puchaseDates! 😮

Reply:
yea, its rather an array keeping track of last 2 purchase dates

Reply:
This makes sense! Also, good variable naming 👍

Reply:
Great changes here in variable naming.